blog/2023/05-14-android-v250/index.mdx

```diff
@@ -0,0 +1,80 @@
+It has been a long while since the last update, but after an extended beta testing period with plenty of bug reports (thanks, by the way!) and subsequent fixes, we're excited to finally announce a new major update to Jellyfin Android.
+
+Version 2.5.0 again focused on improvements to the integrated player, and brings various new features, many bug fixes and a lot of technical maintenance and rewrites.
+
+Read on to learn more!
+
+<!--truncate-->
+
+## Integrated Player Improvements
+
+In this release, we finally introduce the highly requested bitrate selection so that you can transcode to lower resolutions to save your precious mobile data while on the go. The audio and subtitle track selection was reworked and improved multiple times and should be a lot more robust now. Thanks to everyone who reported bugs! The complicated nature of various playback methods, inconsistencies between the server and ExoPlayer and misconceptions about their inner workings caused countless edge cases that could only be discovered through a lot of trial and error.
+The player interface itself now has a back button to close the player and a new playback option was added to pick between hardware and software decoding, if the default hardware decoder causes issues for the currently played video.
+
+In the client settings (which you should make sure to check out) we added the option to pick the initial orientation for video playback and now allow choosing whether you want video with SSA subtitles to be transcoded (for full styling support) or not (to direct play with limited styling). There's also a link to the operating system's subtitle customization settings, which the integrated player uses to style your subtitles. Note that the subtitle settings in the Jellyfin settings only apply to the web player due to technical reasons.
+
+## Other Tweaks
+
+Thanks to @susonthapa, the app now shows a splash screen with the Jellyfin logo on start, instead of the previous, boring gray screen where you didn't see whether the app was actually doing anything. While the app is connecting, there's now a cancel button so that you can quickly jump to the connect screen to select a different server if the current one isn't available at the moment.
+The connect screen itself was reworked and rewritten in JetPack Compose, a new UI framework by Google that allows fast and easy UI development. We plan to use it for more parts of the app in the future, especially for the long-term goal of a completely native app experience.
+We also added a monochrome icon so that Jellyfin will look great on your home screen on supported launchers on Android 13 and above.
+
+## TODO – To mention or not to mention?
+
+- Cronet - very technical and most probably don't care. The percentage of users that deployed QUIC is probably also quite low
+- PGSSUB - a full sentence for just one subtitle codec? Idk..
+- Disabled Safe browsing - only applies to very few users and might confuse/scare others
+
+## Download Now
```

Comment:
I normally place the download section beneath the contributors to highlight those that worked on the release. Also easy to just scroll down to the bottom of the page to find it.